frontend/src/components/object/ObjectList.vue

```diff
@@ -33,20 +35,27 @@ const closeInfo = () => {
   displayInfo.value = null;
 };
 
-// Download
-const multiSelectedObjectIds = computed(() => {
-  return multiSelectedObjects.value.map((o) => o.id);
-});
+const showUpload = () => {
+  displayUpload.value = true;
+};
+
+const closeUpload = () => {
+  displayUpload.value = false;
+};
 
-// Objects List
 const listObjects = async () => {
   try {
-    await objectStore.listObjects({ bucketId: route.params.bucketId });
+    await objectStore.listObjects({ bucketId: route.query.bucketId });
```

Comment:
There may be items in the DB that don't have a bucketId associated to them. If the bucketid is undefined it will just grab every object. Any of these issues should go away once the correct permission checking is in place.